src/locales/README.md

```diff
@@ -0,0 +1,46 @@
+# Contributing a Mirador translation
+
+Mirador is built with internationalization and translation support. It uses a translation file to do this.
```

Comment:
Would it be useful to link to the react-i18n docs too?

Reply:
Great suggestion. I've pushed this up.